azure-pipelines-official.yml

```diff
@@ -65,23 +59,27 @@ jobs:
               -officialBuildId $(Build.BuildNumber)
               -officialSkipTests $(SkipTests)
               -officialSkipApplyOptimizationData $(SkipApplyOptimizationData)
-              -vsDropName $(VisualStudio.DropName)
-              -vsDropAccessToken $(System.AccessToken)
+              /p:RepositoryName=$(Build.Repository.Name)
+              /p:VisualStudioIbcSourceBranchName="master-vs-deps"
```

Comment:
Why are we hard coding this branch name here instead of `$(Build.SourceBranchName)`?

Reply:
Because we only train IBC data from `master-vs-deps` currently, so other branches need to use `master-vs-deps`.

Reply:
Is that for the long term? That's going to fall over pretty soon here. By Friday `master-vs-deps` will be dev16.1. Won't that bust all of the training data we're inserting into dev16.0?

Reply:
We can change this any time we need.

Reply:
This is just a string. It can be whatever we want it to be. It just needs to match to the value that the Release Pipeline that produces the data uses.

Reply:
We will probably want to run the Release Pipeline for 2 branches.

Reply:
In any case, the value will certainly not be `$(Build.SourceBranchName)` because we will only be training from `-vs-deps` branch.

Reply:
How do we control the string which is sent to the release pipeline?

Reply:
That's a parameter in the Release Pipeline definition:

![image](https://user-images.githubusercontent.com/41759/52381633-17a08e00-2a27-11e9-9bcc-8523fadeb673.png)

Reply:
I must be missing something here. That uses `$(SourceBranchName)` which confuses me why we don't use it here.

Reply:
The Release Pipeline is independent from Signed builds. Each signed build ingests some optimization data but only some signed builds kick off IBC release pipeline. 

Consider a feature branch that someone wants to build signed build from. If we used SourceBranchName here it would fail because IBC release pipeline is not running for that feature branch.

Reply:
An alternative approach would be to use `$(SourceBranchName)` here but set `SkipApplyOptimizationData` to true only for select branches.